packages/ckeditor5-watchdog/tests/editorwatchdog.js

```diff
@@ -10,6 +10,35 @@ import Editor from '@ckeditor/ckeditor5-core/src/editor/editor';
 import ClassicTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/classictesteditor';
 import CKEditorError from '@ckeditor/ckeditor5-utils/src/ckeditorerror';
 import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import { Comments } from '@ckeditor/ckeditor5-comments';
+import { TrackChanges } from '@ckeditor/ckeditor5-track-changes';
+import Suggestion from '@ckeditor/ckeditor5-track-changes/src/suggestion';
+
+class UsersInit {
```

Comment:
I've been wondering if we don't have to also "copy users" (the same way as we copy suggestions and comments) to the newly created editor. But for RTC that is not needed and for non-RTC we should not need more users than initially provided.

Reply:
Yes, I was also wondering about it but I realized that we always initialize users by using separate plugins and also it's not a "fresh" issue.